scripts/eval_calc_errors.py

```diff
@@ -276,6 +281,11 @@ def calculate_errors_per_image(im_metaData):
                 depth_im = inout.load_depth(depth_path)
                 depth_im *= scene_camera[im_id]["depth_scale"]  # Convert to [mm].
 
+            # Create im_targets directly from scene_gt and scene_gt_info.
+            im_gt = scene_gt[im_id]
+            im_gt_info = scene_gt_info[im_id]
+            im_targets = inout.get_im_targets(im_gt=im_gt, im_gt_info=im_gt_info, visib_gt_min=p["visib_gt_min"], eval_mode=p["eval_mode"])
```

Comment:
Cf discussion https://github.com/thodan/bop_toolkit/pull/131/#issuecomment-2289536806:
Here we should override im_targets only in the case we are in a bop24/detection mode. This could be simply detected by checking if im_targets is None.

Reply:
@MedericFourmy sounds good, thanks! Could you please run unit tests defined in bop_toolkit_lib/tests to make sure it works properly ?